src/src/components/ObjectBrowser.jsx

```diff
@@ -3578,39 +3609,41 @@ class ObjectBrowser extends Component {
         return [];
     }
 
-    async _exportObjects(isAll, noStatesByExportImport) {
+    async _exportObjects(isAll, noStatesByExportImport, beautifyJsonExport, excludeSystemRepositoriesFromExport, excludeTranslations) {
```

Comment:
Optional: with so many options (and also many optional) we could think about an options object instead